docs/introduction/index.rst

```diff
@@ -19,7 +19,12 @@ Features
 --------
 
 Renku consists of a collection of services, including a web-based user interface
-and a command-line client.
+and a command-line client, exploiting in a coherent setup the joint features of:
+* GitLab - version control
```

Comment:
gitlab is actually for repository management -- version control is handled by git. Could we just say `GitLab - repository management`?